person.rb

```diff
@@ -2,14 +2,15 @@
 
 class Person < Nameable
   attr_reader :id
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
 
   def initialize(age, name = 'unknown', *)
     super()
     @id = rand(0..1_000_000)
     @age = age
     @name = name
     @parent_permission ||= true
+    @rentals = []
   end
```

Comment:
- [ ] This should be the class `Person`, not (Person < Nameable), so please implement the class Person with its functionalities.

Reply:
In the previous project:  "OOP school library: decorate a class", the content is different from the other fellows who are in the same cohort as me. 

This is the link in my progress page: [https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md)

This is the link in my friend's progress page:[https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator.md]( https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator.md)

The project goal is the same only the way of doing it & instructions are different; so that's why my class names and some inheritance adjustments are different. 

But anyway thank you for pointing it out.